pkg/fixtures/writer_s3.go

```diff
@@ -0,0 +1,108 @@
+package fixtures
```

Comment:
this file should now be called writer_blob instead, shouldn't it?

Reply:
Agreed and Applied